README.md

```diff
@@ -11,50 +11,84 @@
 
 ## Setting up for local development
 
-### Prerequisites
+> [!IMPORTANT]  
+> This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance (lucky you!)
 
-This README assumes you have Docker installed. If you do not, please read [this](https://docs.docker.com/engine/install/) manual, based on your operating system. It will cover setting up and testing your very own Docker instance (lucky you!)
+### Requirements
+Node: `7.0.1`
```

Comment:
This should be a Markdown list, and there should be an empty line after headings for MD compatibility.